src/Core/Geographic/Coordinates.js

```diff
@@ -18,6 +19,9 @@ export function ellipsoidSizes() {
 
 const ellipsoid = new Ellipsoid(ellipsoidSizes());
 
+proj4.defs('EPSG:3946',
```

Comment:
I implemented this change in commit b2de20e 
For now I've temporarly moved `proj4.defs(...)` in ApiGlobe, but it could be in `index.html`